docs/configuration/indexes.md

```diff
@@ -211,6 +211,25 @@ credentials cannot be found. If the discovered credentials are not valid (i.e.,
 HTTP 401 or 403), then uv will treat packages as unavailable and query the next configured index as
 described in the [index strategy](#searching-across-multiple-indexes) section.
 
+### Ignoring error codes when searching across indexes
+
+When using the `first-index` strategy, uv will stop searching if it encounters a `401 Unauthorized`
+or `403 Forbidden` response status code. The one exception is that uv will ignore 403s when
```

Comment:
Elsewhere, we stylize error codes as "HTTP XXX" instead of "`XXX NAME`". I do think the name is nice, I might do "HTTP 403 Forbidden" though? I'd probably omit the backticks, since you refer to the error codes in the subsequent text without them.

You can probably just say "status code" instead of "response status code".

Generally, I'd avoid using "it" to refer to uv if it's easy. So... "uv will stop searching if a ... is encountered" rather than "uv will stop searching if it encounters ...".